data-raw/standard_format_markdowns/combine_data_and_eda.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Prey Data Overview"
+author: "Maddee Rubenson (FlowWest)"
+date: "`r Sys.Date()`"
+output: rmarkdown::github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      warning = FALSE, 
+                      message = FALSE)
+
+library(tidyverse)
+library(lubridate)
+library(knitr)
+library(readxl)
+library(sf)
+library(preyDataProcessing)
+
+options(scipen=999)
+```
+
+## Prey Density 
+
+### Aggregate All Prey Data 
+
+```{r}
+
+all_prey_density <- bind_rows(preyDataProcessing::cordoleani_prey_data,
+                      preyDataProcessing::zeug_merced_prey_data, 
+                      preyDataProcessing::zeug_sanJoaquin_prey_data, 
+                      preyDataProcessing::montgomery_prey_data, 
+                      preyDataProcessing::guignard_prey_data, 
+                      preyDataProcessing::corline_prey_data,
+                      preyDataProcessing::zooper_prey_data) 
+
+# usethis::use_data(all_prey_density, overwrite = TRUE)
+
+```
+
+
+### Create Monthly Prey Density Dataset
+
+**Notes:**
+
+* There are two NAs associated with the month of San Joaquin data because data received was aggregated to year
+
+```{r}
+# monthly aggregation by watershed,  habitat type -------------------------
+## group by watershed, habitat type, year, month - summarize prey density (median, min, max)
+monthly_prey_density <- all_prey_density %>%
+  mutate(year = ifelse(!is.na(date), lubridate::year(date), 2016),
+         month = lubridate::month(date)) %>%
+  group_by(watershed, habitat_type, year, month) %>%
+  summarise_at(vars(prey_density), .funs = c('min', 'max', 'median', 'var'), na.rm = TRUE) %>%
+  ungroup() %>%
+  rename(min_prey_density = min,
+         max_prey_density = max,
+         median_prey_density = median,
+         var_prey_density = var) |> glimpse()
+
+# save file for use in R package
+# usethis::use_data(monthly_prey_density)
+
+```
+
+
+<!-- ## Fish Data  -->
```

Comment:
What is happening to the formatting here? Looks kinda crazy. If we are holding off on the fish and environmental I would just put text describing that here so that the documentation can remain legible.